Cargo.toml

```diff
@@ -37,11 +37,11 @@ include = [
 
 [build-dependencies]
 regex = "1"
-bindgen = "0.66.1"
+bindgen = "0.70"
 llvm-tools = { version = "0.1.1", optional = true }
 
 [features]
-default = ["llvm-objcopy"]
+default = ["llvm-objcopy", "nrf9160"]
```

Comment:
While nice for nrf9160 users, this is a slight annoyance for users of other chips. It was a deliberate decision to have it as it was

Reply:
Oversight on my part.